dev-libs/libuninum/libuninum-2.7.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+DESCRIPTION="A library for converting unicode strings to numbers and vice versa"
+HOMEPAGE="https://billposer.org/Software/libuninum.html"
+SRC_URI="https://billposer.org/Software/Downloads/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2"
```

Comment:
No `+`?

Reply:
Hm good question.
The file `numconf.c` states explicit `GPL-2`,
`uninum.c` says `LGPL-2`,
`nsdefs.h` and `uninum.h` says `LGPL-2.1`,
but other files state `GPL-2+`.

Add all 4 licenses?